app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,56 @@
+require 'report_email_sender'
+
+class ReportsController < ApplicationController
+  def new
+    @report = Report.new
+  end
+
+  def create
+    keywords = params.select { |k, v| /keyword_[1|2|3]/.match(k) && !v.empty? }
```

Comment:
is we name the keywords fields `keyword[]` then rails will make an array for us automatically